bin/ci

```diff
@@ -73,13 +73,13 @@ prepare_system() {
 }
 
 build() {
-  with_build_env 'make std_spec clean'
-  with_build_env 'make crystal std_spec compiler_spec docs'
+  with_build_env 'make std_spec clean threads=1'
```

Comment:
Shouldn't `threads` be passed as a flag, i.e. `--threads=1`?

Reply:
Apparently not, the syntax for passing arguments to `make` is that one.